### ISSUE TITLE
C structure/_Complex variable type Intellisense bug!?

### ISSUE THREAD

**Issue by erdiata** [microsoft/vscode-cpptools#1112]
Hello,

In the following code, when I type "mystr.", a list of all members of mystruct_s is listed (a and b) as one would expect. 

```
#include <stdio.h>
#include <complex.h>

typedef struct mystruct_s {
    int a;
    int b;
} mystruct_t;

int main() {
    mystruct_t mystr;
    mystr. // <<<<<----- a list of member variables is displayed
    return 0;
}
```

However,  when I introduce another variable of the type _Complex, no list is displayed!! 

The following code is an example when one of the member variables of the structure mystruct_s is a _Complex data type, the listing of member variables ceases.
```
#include <stdio.h>
#include <complex.h>

typedef struct mystruct_s {
    int a;
    int b;
    double _Complex  c;
} mystruct_t;

int main() {
    mystruct_t mystr;
    mystr. // <<<<<----- no list of member variables is displayed
    return 0;
}
```

**Comment by agauniyal** [comment]
`double _Complex  c;` is valid? 🤔

**Comment by erdiata** [comment]
Part of C99 standard.

**Comment by sean-mcmanus** [comment]
This is related to https://github.com/Microsoft/vscode-cpptools/issues/689 .

**Comment by sean-mcmanus** [comment]
Fix with 0.18.0.
